range.c

```diff
@@ -218,13 +247,13 @@ range_eql(VALUE range, VALUE obj)
 
 /*
  * call-seq:
- *   rng.hash    -> integer
+ *   hash -> integer
  *
- * Compute a hash-code for this range. Two ranges with equal
- * begin and end points (using <code>eql?</code>), and the same
- * #exclude_end? value will generate the same hash-code.
+ * Returns the integer hash value for +self+.
+ * Two range objects +r0+ and +r1+ have the same hash value
+ * if and only if <tt>r0.eql?(r1)</tt>.
  *
- * See also Object#hash.
+ * Related: Range.eql?, Object.hash.
```

Comment:
I think you want `Range#eql?` and `Object#hash` here, since you are referring to instance methods and not class methods.

Reply:
Done.